### DISCUSSION TITLE
Broken character not recognizable

### DISCUSSION THREAD

**Discussion by wazidchoudhary** [naptha/tesseract.js#954] [General]
Hey guyz can someone help on extract text from ocr images of broken character like 
![downloaded_image - 2024-09-17T000722 019](https://github.com/user-attachments/assets/5ed03ac5-9c38-435a-9e72-d3706f234c0d)
![downloaded_image - 2024-09-17T003654 506](https://github.com/user-attachments/assets/4ce1f2b4-2ab2-4bc5-8933-193e88df8fda)

**Comment by Balearica** [comment]
When recognizing low-quality scans, the results will contain errors.  This will be true for every OCR program, and there is no turn-key solution (if there were, it would be enabled by default).  If you need high-quality text extraction from low-quality documents, then you would need to have a human reviewer correct errors using a tool like [Scribe OCR](https://scribeocr.com/).
